apps/core/test/test_helper.exs

```diff
@@ -1 +1,4 @@
 ExUnit.start()
+
+Mix.Task.run "ecto.create", ["--quiet"]
```

Comment:
This will cause CI to fail as the DB is already provisioned by Heroku: https://til.energizedwork.com/posts/6iwjc0hsjy-heroku-ci-config-for-elixir-apps